web/src/config/client_swagger.json

```diff
@@ -54,34 +36,25 @@
 		"/login": {
 			"post": {
 				"tags": ["User"],
-				"summary": "Admin User, CSM user and S3 User login",
+				"summary": "This API allow login to Admin user, CSM user and S3 User",
 				"responses": {
 					"200": {
-						"description": "OK"
-					},
-					"201": {
-						"description": "No Content"
+						"description": "OK: User logged in successfully."
 					},
 					"400": {
-						"description": "Bad Request"
+						"description": "Bad Request: Request body is missing or Username or password is missing"
 					},
-					"404": {
-						"description": "No Data Found"
-					},
-					"409": {
-						"description": "Conflict"
+					"401": {
+						"description": "Unauthorized: Token has not found or invalid format or session is invalid"
```

Comment:
For login , "username and password combination is in correct",  Session and token generation happens in this api.

Reply:
Updated message with correct text